src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -126,6 +127,8 @@
 public class PlayerMock extends HumanEntityMock implements Player, SoundReceiver
 {
 
+	private final Component defaultKickComponent = Component.text("You are not whitelisted on this server!");
```

Comment:
```suggestion
	private static final Component DEFAULT_KICK_COMPONENT = Component.text("You are not whitelisted on this server!");
```
Unless there's a setter to change this (which I don't think is necessary) then there's no reason for it to not be a proper constant.